templates/bridge-infra/agglayer-config.toml

```diff
@@ -18,8 +18,9 @@ mock-verifier = {{.mock_verifier}}
 [proof-signers]
 # OP Sovereign Admin Address
 {{- if and .deploy_optimism_rollup (ne .consensus_contract_type "fep") }}
-# OP Sovereign Admin Address
-1 = "{{.zkevm_l2_sovereignadmin_address}}"
+1 = "{{.zkevm_l2_sequencer_address}}"
+{{- else if and .deploy_optimism_rollup (eq .consensus_contract_type "fep") }}
+1 = "{{.zkevm_l2_sequencer_address}}"
 {{- else }}
 1 = "{{.zkevm_l2_sequencer_address}}"
 {{- end }}
```

Comment:
I think we should set `[proof-signers]` to `zkevm_l2_sequencer_address`. We can probably remove all the conditions in this section - what do you think, @jhkimqd?